### DISCUSSION TITLE
Renovate temporary-error from executionTimeout should be reported to Dependency Dashboard

### DISCUSSION THREAD

**Discussion by KreisichD** [renovatebot/renovate#31982] [Suggest an Idea]
## Description
I think that Renovate should report at least some of the temporary-error's to Dependency Dashboard. Specifically I'm pointing on the timeout as it might not be temporary and in fact there can be a dependency clash between different renovate branches resulting in Renovate failing without notice for repository owners.

Explanation of situation:
1. Project is using `poetry`, with lots of dependencies, without strict requirements
2. Update of some dependencies cannot be performed due to conflict (one package in newer version have incompatible sub-dependencies with other package)
3. Renovate fails on `poetry upgrade --lock <package-name>` due to executionTimeout (process would probably never finish anyway, so timeout is valid)
4. Nothing is reported to Dependency dashboard, no related MR's are being created etc. 
5. User has to find out about that himself and look for logs later

While it might not be very frequent situation, it often might be cause by unfortunate grouping and other configuration parts, technology etc. I feel like it might be an upgrade in terms of user experience.

Most likely discussions like this https://github.com/renovatebot/renovate/discussions/22090 would not be required if it was clearly communicated that the timeout did happen, on what command etc.

## How can that be implemented
I feel like it could be implemented using different kind of error i.e execution-timeout
Then the error can be handled on any level in the Renovate execution stack, since it is specific to all of the execution-timeout related Renovate failures. Any information would probably be better than none in such case.

Such information should most likely be visible in dependency dashboard, as MR might not exist. 

## Similar discussions
https://github.com/renovatebot/renovate/discussions/29961

Please let me know what do you think

**Comment by rarkins** [comment]
Good idea, converted to an issue: https://github.com/renovatebot/renovate/issues/31993

PR welcome if you'd like to contribute it
